assets/course-theme/course-theme.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { toggleFocusMode } from './focus-mode';
+
+if ( ! window.sensei ) {
```

Comment:
Introducing the `sensei` global for namespacing global frontend JS.

Reply:
Good!

This might be interesting also for Nexus right?